src/status_im/contexts/chat/home/new_chat/view.cljs

```diff
@@ -49,13 +49,27 @@
                                (rf/dispatch [:open-modal :new-contact]))}
       (i18n/label :t/add-a-contact)]]))
 
+(def ^:private max-group-chat-contacts (dec constants/max-group-chat-participants))
```

Comment:
As we have `selected-contacts-count` for comparison, should we use something like `contacts-selection-limit`?